src/server.js

```diff
@@ -22,11 +24,28 @@ const connected = false;
 const app = express();
 app.use(express.static(`${__dirname}/public`));
 
+// Setup exception logging
+const applicationMode = process.env.ENVIRONMENT;
+if (applicationMode === 'production' || applicationMode === 'staging') {
+    Sentry.init({
+        dsn: 'https://8ae89827146d49309d5a8e5699a605b6@sentry.io/1419602',
+        environment: applicationMode
+    });
+}
+
 // Redirect default '/' call to main.htm page
 app.get('/', (req, res) => {
     res.redirect('/index.html');
 });
 
+console.log("Start waiting....");
+if (process.env.IS_BACKUP === "true") {
+    sleep.sleep(10);
+} else {
+    sleep.sleep(15);
+}
+console.log("Stop waiting....");
+
 // Start listening with HTTP (picks random available port)
 const server = http.createServer(app).listen(8000);
```

Comment:
Please consider turning the port into a constant.

Reply:
That is outside of the scope of the enhancements done in this PR.